### PR TITLE
Added explainer for optimize image policies

### PR DESCRIPTION
**Pull request by loonybear**
Hi iclelland@, PTAL. Thanks!

### CODE REVIEW

**Comment by clelland** [comment]
Marked as non substantive for IPR from ash-nazg.

**Comment by yoavweiss** [comment]
@clelland The contribution seems substantive...
The right thing to do here is for @loonybear to join the WICG.

cc @cwilso

**Comment by clelland** [comment]
(See https://github.com/WICG/feature-policy/pull/155#issuecomment-385419177 as well)

This isn't normative to the feature policy spec, though it is a substantial piece of work that may generate a lot of discussion. And agreed that it's probably a good idea for @loonybear to join anyway.

File: policies/optimized-images.md [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+
+Optimized image policies introduce a set of restrictions (policies) on images that can be applied with dev-time enforcement. An image will be rendered with **inverted colors** when violating a policy, making it easy for web developers to identify and fix the error.
+
+
+### Optimized image policies
+
+*   **["legacy-image-formats" policy](#legacy-image-formats)**
+    *   Images must be of one of the _**modern formats***_ (JPEG, PNG, WEBP, etc).
+*   **["maximum-downscaling-image" policy](#maximum-downscaling-image)**
+    *   Images must not be bigger than its container size by more than _**X times***_ .
```
Comment (igrigorik) [inline policies/optimized-images.md]: How do we reason about device DPR? Many sites today provide "2x" resources in .. based on assumption that most mobile devices are at least 2x.

File: policies/optimized-images.md [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+*   **["image-compression" policy](#image-compression)**
+    *   Images used in rendering must not include too much metadata.
```
Comment (igrigorik) [inline policies/optimized-images.md]: Can we define what too much is? :)

File: policies/optimized-images.md [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+</a>
+
+Image formats affect file size, loading speed and appearance. Modern image formats yield large byte savings and performance improvement. `legacy-image-formats` is a policy controlled feature that restricts images to be one of certain modern formats.
+
+When a document is disallowed to use `legacy-image-formats` policy, its `<img>` elements will render images of "legacy" formats with inverted colors.
+
+
+#### Specification
+
+- The default list of modern image formats is: JPEG, PNG, GIF, WEBP, and SVG.
```
Comment (igrigorik) [inline policies/optimized-images.md]: GIF, really? When's the right time to use GIF on the web? If anything, seems like we should be pushing folks aggressively away from GIF to video.

**Comment by annevk** [comment]
@igrigorik I recommend filing issues as this is already merged.

**Pull request merged**